src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfBalls = Lottery.getBallsAmount();
```

Comment:
It is not convenient to declare the number of balls within the Lottery, because the responsibility of the Lottery is to generate random balls. So let's not use it and simply declare the number of balls as a constant `3`.